### PR TITLE
allan updates

### PR DESCRIPTION
**Pull request by n9d0g**

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
 	"name": "web",
 	"private": true,
-	"version": "3.15.9",
```
Comment (n9d0g) [inline apps/web/package.json:4]: we should keep this in, it helps us keep track of what version we're on

File: apps/web/src/app.html [hunk 2]
```diff
@@ -24,12 +37,21 @@
...
+	<body
+		data-sveltekit-preload-data="hover"
+		data-theme="fcc-theme"
+		class="realative"
+	>
+		<div class="absolute left-0 top-0 z-[1] h-72 w-full opacity-80">
+			<div class="absolute inset-0" id="particles-js"></div>
 		</div>
+
+		<div style="display: contents" class="z-[1]">%sveltekit.body%</div>
```
Comment (n9d0g) [inline apps/web/src/app.html:49]: why did you remove the `h-full overflow-hidden` classes here? and what is the `z-[1]` class? 

also please try to keep the `bg-surface-300-600-token` class here as it helps with the light/dark theme background colours

File: apps/web/src/app.html [hunk 3]
```diff
@@ -24,12 +37,21 @@
...
+
+		<script>
+			particlesJS.load('particles-js', 'data/particles.json', function () {
+				console.log('callback - particles.js config loaded')
+			})
+		</script>
```
Comment (n9d0g) [inline apps/web/src/app.html:55]: i think this `<script>` can be imported as an NPM package and then applied elsewhere -- i'm not too sure this belongs in the `app.html` file

File: apps/web/src/routes/ministries/+page.ts [hunk 4]
```diff
@@ -4,41 +4,58 @@ export const load = async () => {
...
 			},
 			{
 				title: 'Youth',
 				link: '/ministries/youth',
+				icon: '/images/youth-icon.png',
+				backgroundImage: '/images/youthImage.jpg',
 			},
 			{
-				title: 'Young Adults',
+				title: 'Young People',
```
Comment (n9d0g) [inline apps/web/src/routes/ministries/+page.ts:55]: pls change back to "Young Adults"

File: apps/web/src/routes/events/+page.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@ export const load = async () => {
 	const breadcrumb = [breadcrumbs.home, breadcrumbs.events]
 
 	return {
-		title: 'FCC Events.',
+		title: '',
```
Comment (n9d0g) [inline apps/web/src/routes/events/+page.ts:7]: let's try to keep things consistent here -- if you want to change the `h1` to "FCC Sunday Specials", then you can add that here instead of adding an `h1` to the `Holidays` component

File: apps/web/src/lib/components/home/HomeWelcome.svelte [hunk 6]
```diff
@@ -3,25 +3,39 @@
...
-		data-testid="welcome-subheader"
-	>
-		We welcome you to our worship services every Sunday morning at 10:30am!
-		Services will be both online (Zoom) and in-person. Click the button below to
-		join via Zoom, or join us in-person at
-		<a href={links.googleMaps} target="_blank" class="anchor font-bold">
-			5225 Orbitor Drive Unit 3, Mississauga, ON.
-		</a>
-	</p>
+	<div class=" w-full mt-10 text-white rounded-lg bg-slate-800 p-8">
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/HomeWelcome.svelte:8]: for this site, let's try to stick with a few colours that go with the colours of the logo.

so it's more green, blue, and white -- let's try not to use `slate-` and all these other colours so that we stay consistent

File: apps/web/src/lib/components/home/HomeWelcome.svelte [hunk 7]
```diff
@@ -3,25 +3,39 @@
...
+		<h2
+			class="h2 mb-6 text-center text-4xl font-bold"
+			data-testid="welcome-header"
+		>
+			Come join us!
+		</h2>
+		<p
+			class="mx-auto text-center text-2xl tracking-widest md:max-w-[50rem]"
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/HomeWelcome.svelte:16]: i'm not really liking the `tracking-widest` from a design perspective, i think it looks better without the `tracking`

File: apps/web/src/lib/components/home/HomeWelcome.svelte [hunk 8]
```diff
@@ -3,25 +3,39 @@
...
+			data-testid="welcome-subheader"
+		>
+			We welcome you to our worship services:
+		</p>
+		<ul class="mt-6 rounded-lg border p-4 text-center text-2xl">
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/HomeWelcome.svelte:21]: you use `<ul>` elements when you have a list of things, ie.

- list item 1
- list item 2

i don't think you need to use a `<ul>` here, you can just use `<p>` tags

Comment (n9d0g) [reply apps/web/src/lib/components/home/HomeWelcome.svelte:21]: `<ul>` stands for "Unordered list", and `<li>` stands for "List item"

File: apps/web/src/lib/components/home/HomeWelcome.svelte [hunk 9]
```diff
@@ -3,25 +3,39 @@
...
+			<li>
+				every Sunday morning at 10:30am! Services will be both online (Zoom) and
+				in-person.
+			</li>
+			<li>Click the button below to join via Zoom, or join us in-person at</li>
+			<li>
+				<i class="fa-sharp fa-regular fa-map"></i>
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/HomeWelcome.svelte:28]: i think we can remove this map icon, i'll find something better to use

File: apps/web/src/lib/components/home/Hero.svelte [hunk 10]
```diff
@@ -1,25 +1,53 @@
...
-		</h2>
-		<div class="my-8 flex gap-8">
-			<a
-				href="/about"
-				class="bg-surface-200-700-token text-token btn btn-xl"
-				data-sveltekit-preload-data="hover">I'm new</a
+
+<section style="background-image: url('/images/youngPeopleImage.jpg');" class="flex h-[30vh] sm:h-[90vh] bg-contain bg-no-repeat sm:bg-cover sm:bg-gradient-to-t sm:from-blue-950 to-green-800 items-center justify-center ">
+	
+	<div style="background-image: url('/images/youthImage.jpg');" class="sm:rounded-2xl sm:bg-gradient-to-t sm:from-blue-950 to-green-800 mt-40 sm:mt-[480px] mx-auto bg-contain bg-no-repeat flex h-[420px] sm:h-auto p-4 items-center justify-around ">
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/Hero.svelte:4]: i like the `<section>` background, but i think adding another image on top of that is a little too much. We can probably remove this image and then style the rest of the text instead

File: apps/web/src/lib/components/home/Hero.svelte [hunk 11]
```diff
@@ -1,25 +1,53 @@
...
+		<div
+			class="container mx-10 flex h-full flex-col items-center justify-center"
+		>
+			<h1
+				class="tracking-widest mt-80 sm:mt-[300px] h1 mb-10 flex flex-col rounded-lg sm:border sm:bg-green-800 p-2 text-center text-5xl font-bold text-white sm:shadow-md sm:shadow-white"
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/Hero.svelte:9]: we can remove the `tracking-widest`, also we can use a softer green colour instead -- this one seems a little too strong. I'll probably make some more styling changes after this PR gets approved

File: apps/web/src/lib/components/home/Hero.svelte [hunk 12]
```diff
@@ -1,25 +1,53 @@
...
 			>
-			<a
-				href="/sermons"
-				class="bg-surface-200-700-token text-token btn btn-xl"
-				data-sveltekit-preload-data="hover">Sermons</a
+				Freedom in Christ <span class=" text-white">Church</span>
+			</h1>
+			<h2
+				class="h2 hidden sm:flex top-52 sm:mt-[30px] border-b border-slate-700 text-center text-2xl text-white"
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/Hero.svelte:14]: you can use just one `<h2>` tag here, and then break it up with `<br />` elements. Also why is this `hidden` on mobile? I think we also want to show this text on mobile

File: apps/web/src/lib/components/home/Hero.svelte [hunk 13]
```diff
@@ -1,25 +1,53 @@
...
+			class="bg-surface-200-700-token text-token btn btn-xl border"
+			data-sveltekit-preload-data="hover">Sermons</a
+		>
 	</div>
 </section>
+<div
+	class="sticky top-0 z-20 flex justify-center border-y bg-warning-800 p-2 shadow-2xl shadow-green-700"
+>
+	<button>
+		<h1 class="text-4xl tracking-widest">Schedule your first visit!</h1>
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/Hero.svelte:51]: There should only be one `<h1>` on a page at a time, you can remove this and just add the styling to the `<button>` element

File: apps/web/src/lib/components/home/Hero.svelte [hunk 14]
```diff
@@ -1,25 +1,53 @@
...
+		>
+		<a
+			href="/sermons"
+			class="bg-surface-200-700-token text-token btn btn-xl border"
+			data-sveltekit-preload-data="hover">Sermons</a
+		>
 	</div>
 </section>
+<div
+	class="sticky top-0 z-20 flex justify-center border-y bg-warning-800 p-2 shadow-2xl shadow-green-700"
```
Comment (n9d0g) [inline apps/web/src/lib/components/home/Hero.svelte:48]: I really like the sticky idea here, but let's use a different colour instead of `warning`

File: apps/web/src/lib/components/home/Hero.svelte (hunk 12 above)
Comment (n9d0g) [reply apps/web/src/lib/components/home/Hero.svelte:14]: it's also not the best on desktop in front of the background image - very hard to read. I know it's just a placeholder, but just something to think about

File: apps/web/src/lib/components/holidays/holidays.svelte [hunk 15]
```diff
@@ -0,0 +1,44 @@
+<script>
+	import HolidayCard from '$lib/components/HolidayCard.svelte'
+</script>
+
+<h1 class="my-4 text-4xl">FCC Sunday Specials</h1>
+<div
```
Comment (n9d0g) [inline apps/web/src/lib/components/holidays/holidays.svelte:6]: we can loop through these data points, or also hook it up to the CMS

**Comment by allanReyMacasaquit** [comment]
Thanks for the feedback Nathan, it's my mistake you had to go through these
amateur moves. lol!
I will be careful next time. Obviously I need to learn more.. Have a good
day.

**Comment by n9d0g** [comment]
@allanReyMacasaquit no worries 😃 let's tackle one problem at a time and we'll work from there 👍
